platform/darwin/scripts/generate-style-code.js

```diff
@@ -4,11 +4,20 @@ const fs = require('fs');
 const ejs = require('ejs');
 const _ = require('lodash');
 const colorParser = require('csscolorparser');
-const spec = _.merge(require('mapbox-gl-style-spec').latest, require('./style-spec-overrides-v8.json'));
-
+const cocoaConventions = require('./style-spec-cocoa-conventions-v8.json');
+let spec = _.merge(require('mapbox-gl-style-spec').latest, require('./style-spec-overrides-v8.json'));
 const prefix = 'MGL';
 const suffix = 'StyleLayer';
 
+// Rename properties and keep _original for use with setters and getters
+_.forOwn(cocoaConventions, function (properties, kind) {
+    _.forOwn(properties, function (newName, oldName) {
+        spec[kind][newName] = spec[kind][oldName];
+        spec[kind][newName]["_original"] = oldName;
```

Comment:
Ack, sorry, I meant `._original`. You were right about trying to avoid a theoretical conflict with a property named `original`.